app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -33,26 +35,36 @@ class FileSetsController < ApplicationController
     # GET /concern/file_sets/:id
     def edit
       initialize_edit_form
+    rescue WorkflowAuthorizationException
+      render_unavailable
     end
 
     # GET /concern/parent/:parent_id/file_sets/:id
     def show
+      workflow_check(parent_id: params[:parent_id])
       respond_to do |wants|
         wants.html { presenter }
         wants.json { presenter }
         additional_response_formats(wants)
       end
+    rescue WorkflowAuthorizationException
```

Comment:
as in https://github.com/samvera/hyrax/pull/3881/files#r305449920

Reply:
It is. I had problems with this at first and thought it was a bug with 'rescue_from' (which I researched), but it isn't. Removed.